google-datacatalog-rdbms-connector/tests/google/datacatalog_connectors/rdbms/prepare/datacatalog_tag_factory_test.py

```diff
@@ -226,13 +226,11 @@ def test_make_tags_for_columns_metadata_should_set_all_available_fields(
             make_tags_for_columns_metadata(tag_template, tables_dict)
 
         tag_1 = tags[0]
```

Comment:
```suggestion
        tag = tags[0]
```
No need to use the suffix since only 1 tag will be created.